templates/admin/auth/source/oauth.tmpl

```diff
@@ -24,7 +24,7 @@
 		<label for="oauth2_icon_url">{{.locale.Tr "admin.auths.oauth2_icon_url"}}</label>
 		<input id="oauth2_icon_url" name="oauth2_icon_url" value="{{.oauth2_icon_url}}">
 	</div>
-	<div class="open_id_connect_auto_discovery_url required field">
+	<div class="open_id_connect_auto_discovery_url required field{{if .Err_DiscoveryURL}} error{{end}}">
```

Comment:
```suggestion
	<div class="open_id_connect_auto_discovery_url required field {{if .Err_DiscoveryURL}}error{{end}}">
```
As I've understood, we've now agreed not save this one space anymore to make it more readable.

Reply:
I don't think we specifically agreed to one form, but accept both forms. I still prefer the "correct" form :)

Reply:
You mean the form without the extra space?
Same.
However, I can also see that it makes reading the code much harder.

> I don't think we specifically agreed to one form, but accept both forms

https://docs.gitea.io/en-us/guidelines-frontend/#styles-and-attributes-in-go-html-template

Reply:
Yes, I prefer the form that does not emit whitespace. It's slightly less readable, but still acceptable to me to not have ugly HTML in the inspector.

I see it was recommended, but that doesn't mean the other form is strictly forbidden.